src/components/GlMap.svelte

```diff
@@ -132,6 +132,7 @@
     map = new glLibrary.Map({
       container: id,
       style: url,
+      canvasContextAttributes: { preserveDrawingBuffer: true },
```

Comment:
Technically we remount if changing the renderer so we could do either/or here, but it seems harmless to just provide both versions of the arg all the time. Let me know if you disagree.